main/webapp/modules/core/scripts/views/data-table/cell-ui.js

```diff
@@ -41,6 +41,8 @@ function DataTableCellUI(dataTableView, cell, rowIndex, cellIndex, td) {
   this._render();
 }
 
+var defaultChoice = true;
```

Comment:
Since you are introducing this as a global variable, I think it would be better to use a more specific name, making it clear which choice it affects. For instance `reconMatchSilimilarCellsByDefault` or something like that.

Reply:
Well noted